homeassistant/components/automation/__init__.py

```diff
@@ -38,13 +38,15 @@
 CONF_ACTION = 'action'
 CONF_TRIGGER = 'trigger'
 CONF_CONDITION_TYPE = 'condition_type'
+CONF_INITIAL = 'initial'
```

Comment:
Not sure if I like this config key, it is ambiguous. What about `initial_state`.

Since YAML translates on/off to True/False, people could write `initial_state: on`